library/sha512.c

```diff
@@ -568,6 +578,10 @@ static const unsigned char sha512_test_sum[6][64] =
       0x4E, 0xAD, 0xB2, 0x17, 0xAD, 0x8C, 0xC0, 0x9B }
 };
 
+#define SHA512_TEST_SUM_N \
```

Comment:
Minor: I'd prefer to write `ARRAY_LENGTH(sha512_test_sum)` with a generic `ARRAY_LENGTH` macro so that it's evident at the point of use that we're looking at the length of the relevant array.